.gitpod.yml

```diff
@@ -0,0 +1,7 @@
+tasks:
+  - name: Initialize Poetry Environment
+    before: pyenv install 3.9.6 && pyenv global 3.9.6
+    env:
+      PIP_USER: false
```

Comment:
Is this for Gitpod itself?

Reply:
This is so poetry can install deps, user-site packages aren't supported